bokeh/document.py

```diff
@@ -346,6 +347,10 @@ def to_json_string(self):
             'roots' : {
                 'root_ids' : root_ids,
                 'references' : self._references_json(root_references)
+            },
+            'version' : {
+                'full' : __version__,
+                'base' : __base_version__
```

Comment:
Tho maybe that's no longer true and it can just be deleted altogether:
```
bryan@10 (git:master) ~/work/bokeh $ grin base_version
./bokeh/__init__.py:
   17 : from .util.version import __base_version__; __base_version__
./bokeh/tests/test_bokeh_init.py:
   12 :         self.assertTrue("__base_version__" in names)
./bokeh/util/version.py:
   31 : def base_version():
   36 : __base_version__ = base_version()
   37 : del base_version
```
@damianavila do you have any info?